pyproject.toml

```diff
@@ -0,0 +1,47 @@
+[tool.poetry]
+name = "netbox-loadtest"
+version = "2.0.1"
+description = "A load test script for the netbox IPAM solution."
+authors = ["Jarrod J Manzer <jjmanzer@godaddy.com>"]
+license = "MIT"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Web Environment",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+]
+packages = [
+  { include = "bin/*.py" },
+]
+
+[[tool.poetry.source]]
+name = "GitHub"
+url = "https://github.com/jjmanzer/netbox-loadtest/"
+
+[tool.poetry.scripts]
+scripts = ["bin/netbox_loadtest.py"]
```

Comment:
I don't think this is the right format for this section... what is this intended to do?

Reply:
to create an executable script in the path upon install that is to be run by the user.